sys-fs/xfsprogs/xfsprogs-9999.ebuild

```diff
@@ -0,0 +1,192 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic systemd udev
+
+DESCRIPTION="XFS filesystem utilities"
+HOMEPAGE="https://xfs.wiki.kernel.org/ https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	inherit autotools
+	REGEN_BDEPEND="
+		dev-build/autoconf
+		dev-build/automake
+		dev-build/libtool
+	"
+	EGIT_REPO_URI="https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git"
+	EGIT_BRANCH="for-next"
+	EGIT_CLONE_TYPE="shallow"
+else
+	REGEN_BDEPEND=""
+	SRC_URI="https://www.kernel.org/pub/linux/utils/fs/xfs/${PN}/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="icu libedit nls selinux static-libs"
+
+RDEPEND="
+	dev-libs/inih
+	dev-libs/userspace-rcu:=
+	>=sys-apps/util-linux-2.17.2
+	icu? ( dev-libs/icu:= )
+	libedit? ( dev-libs/libedit )
+"
+DEPEND="
+	${RDEPEND}
+	>=sys-kernel/linux-headers-6.11
+"
+BDEPEND="
+	${REGEN_BDEPEND}
+	nls? ( sys-devel/gettext )
+"
+RDEPEND+=" selinux? ( sec-policy/selinux-xfs )"
+
+call_prepare_release() {
+	local version
+
+	#test sed -i
+	#	's/^AC_INIT\(\[xfsprogs\],\[([0-9]+).([0-9]+).([0-9]+)\],\[linux-xfs@vger\.kernel\.org\]\)$/\1.\2.9a999/'
+	#	configure.ac
+
+	version="$(grep -o -- \
+		'^AC_INIT(\[xfsprogs\],\[[0-9]\+\.[0-9]\+\.[0-9]\+\],\[linux-xfs@vger.kernel.org\])$' \
+		configure.ac)"
+	if [ $? != 0 ]; then die; fi
+
+	#test version+='a'
+	version=$(echo "${version}" | ( sed -r \
```

Comment:
For this pattern, you probably want `$(echo ... | sed ; pipestatus || die)`) instead and `inherit eapi9-pipestatus`.